jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -436,7 +436,7 @@ def step900_collect_driver_artifacts() {
 }
 
 def create_params_file(archive_dir) {
-    def keys = params.keySet()
+    def keys = params.keySet().toArray()
```

Comment:
Are `params` a groovy map? Can we call these methods on it? http://groovy-lang.org/groovy-dev-kit.html#_iterating_on_maps

Reply:
I don't know if `params` is a map, but from the groovy console in Jenkins it looks like I can use those methods on maps.
http://puppetserver-perf-driver55.delivery.puppetlabs.net:8080/script

Reply:
Nope! Jenkins doesn't support it, I found out yesterday >.>
https://github.com/puppetlabs/gatling-puppet-load-test/pull/208#issuecomment-375123105

Reply:
That's terrifying

Reply:
More reason to upgrade our Jenkins install, I guess? It looks like it was fixed awhile back but I think requires Jenkins 2.7+

https://wiki.jenkins.io/display/JENKINS/Pipeline+Groovy+Plugin#PipelineGroovyPlugin-2.33(May30,2017)